docs/data/routes/connected-demo/explore-sitecore/experience-editor/en.md

```diff
@@ -5,39 +5,31 @@ title: Experience Editor
 ---
 
 # Experience Editor
-Navigate to the Sitecore log-in screen and enter the login credentials provided in your confirmation email.
 
-<p>
-  <div class="row">
-    <div class="col-md-6"> 
-      <p><img src="/assets/img/ExperienceEditor1.jpg" alt="Habitat Fitness"></p>
-    </div>
-    <div class="col-md-6"> 
-      <p>Enter the Experience Editor from the Launchpad.</p>
-      <p>The Experience Editor is a WYSIWYG (What You See Is What You Get) editor that allows you to easily make changes to items directly on the page. You can edit all the items that are visible on the page — text, graphics, logos, links, and so on. In the Experience Editor, you can edit the fields of the current item and the fields of any items rendered on the page. For instance, if the menu on the page consists of titles of the product items, you can edit the titles without navigating to the product items themselves.</p>
-    </div>
-  </div>
-<p>
+The Experience Editor is a WYSIWYG (What You See Is What You Get) editor that allows you to easily make changes to items directly on the page. You can edit all the items that are visible on the page — text, graphics, logos, links, and so on. In the Experience Editor, you can edit the fields of the current item and the fields of any items rendered on the page. For instance, if the menu on the page consists of titles of the product items, you can edit the titles without navigating to the product items themselves.
```

Comment:
The markup was in there to render the image and text side by side, as 2 columns. IIRC, this way looked better or made it easier to understand what text goes with the image.

I'm not sure if that's still needed, but wanted to provide the context.

Reply:
Thanks for the context @anastasiya29! @jflheureux  will review your comment

Reply:
I removed most of these side-by-side images and text in favor of numbered lists with images correctly aligned under each list item. Most of these side-by-side image/text were for instructions to follow in a specific order.

For this one in particular, the Launchpad "Experience Editor" button that was previously instructed to click on is not loading the correct site anymore so I had to change the instructions to go in the Content Editor first.